### ISSUE TITLE
Slots directories that I have no control over.

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#956]
**Reported by Ageless on 23 Apr 40109030 12:53 UTC**
While doing a complete uninstall of BOINC, I found I have a couple of slots directories on two computers that I cannot get rid of. When trying to delete them manually - even after I have completely uninstalled BOINC and managed to get rid of the rest of the Data directory - I always get the following message: "Can't delete 'filename' or map. Access is denied. Check that your disk isn't full or write protected or that the file isn't in use at this moment". 

In one of the places it's a file that won't be removed. I cannot change its attribute due to permission problems. I cannot assume permission over it either. In the other case it's two maps that I cannot access and see what's in them, nor delete. 

Now this may well be a Windows (XP SP3) problem, but I would think that BOINC should at all times have ownership of all files and folders in the Data directory. And since I am a user of the BOINC administrative groups, I should be able to delete the file or folder. 

As far as I can see, the slots were used by either Drugdiscovery@home.com, or Hydrogen@Home. But how can a project take ownership of the slots directories? Isn't that a form of hijacking? And wouldn't that be a security problem?

**Comment by romw** [comment]
**Commented by Ageless on 3 Jul 40109360 09:20 UTC**
I finally managed to get rid of the directories by taking (forced) ownership of the maps and in one case of the one file in them. On another computer I resorted in reformatting the partition with the affected map on it. 

Rom asked if I used my cleaning script on both computers; I didn't. I only used it on one of them, the other was unaffected by it. Besides, the timestamps on the slots folders was of June 2008, so they've been there for a while already. 

Checking the owners of one of the affected maps, I noticed they were boinc_project, None and Everyone. All with only read and write permission. Weird.
